src/main/java/net/sf/jabref/event/ChangedFieldEvent.java

```diff
@@ -0,0 +1,37 @@
+package net.sf.jabref.event;
+
+import net.sf.jabref.model.entry.BibEntry;
+
+/**
+ * <code>ChangedFieldEvent</code> is fired when a field of <code>BibEntry</code> has been modified.
+ */
+public class ChangedFieldEvent {
```

Comment:
Please extend ChangedEntryEvent and do the necessary modifications (calling super, removing the local bibEntry variable and removing getBibEntry).